### DISCUSSION TITLE
Remove goals and indicators

### DISCUSSION THREAD

**Discussion by SandraEliz** [open-sdg/open-sdg#2040] [Q&A] [answered]
Hello

I want to remove indicators and targets from the list that appears in each objective, I already modified the global_indicators.yml and global_targets.yml files, I also deleted their yml files from the indicator-config folder, but they still appear.

What else should be changed?

Thank you

**Comment by SandraEliz** [comment]
Hello

I'm still in the same situation

I already changed the site_config setting

![image](https://github.com/open-sdg/open-sdg/assets/117297044/7bca178f-7adf-4b8a-80b0-15a7e968da22)

What do I need to do?

Doesn't remove targets and indicators

**Comment by brockfanning** [comment]
@SandraEliz If I understand correctly, you are trying to remove particular indicators from your platform. Indicators are automatically created based on three things:

1. Indicator configuration
2. Indicator data
3. Indicator metadata

**If _any_ of these three things exist, then the indicator will be automatically created.** You mention that you deleted the indicator configuration files. You may also need to delete the data and metadata files. In short, to delete an indicator, you need to delete three files.

For your information, the translations do not affect the automatic creation of indicators. So, you don't need to worry about changing the translations.

If this still doesn't work for you, and you still have questions, a link to your repositories would be helpful.

**Comment by SandraEliz** [comment]
Hello

I still have the problem.

I already deleted the files you mention, but the link still appears.

This is the data repository
https://github.com/prep-odsmexico/data

An example is 1-3-1
https://prep-odsmexico.github.io/site/1/

![image](https://github.com/open-sdg/open-sdg/assets/117297044/aeb641a9-f98e-45cc-8964-65ee2e6ad1ab)


Thank you

**Comment by brockfanning** [comment, marked as answer]
@SandraEliz My suspicion is that this is caused by the input that is pulling global metadata. Can you please try removing these two lines, to see if this fixes the problem? https://github.com/prep-odsmexico/data/blob/develop/config_data.yml#L66-L67

**Reply by SandraEliz** [reply]
Hello.

Thank you very much for the reply

It works now
